VirtoCommerce.CatalogModule.Web.Core/Model/Product.cs

```diff
@@ -329,5 +329,67 @@ public virtual CatalogProduct ToModel(CatalogProduct product)
         {
             return product;
         }
+
+        public virtual void ReduceDetails(string responseGroup)
+        {
+            var productResponseGroup = EnumUtility.SafeParseFlags(responseGroup, ItemResponseGroup.ItemLarge);
+
+            if (!productResponseGroup.HasFlag(ItemResponseGroup.ItemAssets))
+            {
+                Assets = null;
+            }
+
+            if (!productResponseGroup.HasFlag(ItemResponseGroup.ItemProperties))
+            {
+                Properties = null;
```

Comment:
Clear PropertyValues.
In general it should be like before for model: https://github.com/VirtoCommerce/vc-module-catalog/pull/226/files#diff-a739380ca84d772b1756c55fdcbec922L143